ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-ref.yaml

```diff
@@ -12,6 +12,9 @@ ref:
   - namespace: test-credentials
     name: ocp-qe-gcp-sa
     mount_path: /ga-gsheet
+  - namespace: test-credentials
+    name: perfscale-metal-bastion
```

Comment:
@jtaleric Where is this saved in vault?

Reply:
@qiliRedHat in a new vault secret that @josecastillolema and I have access to -- do you need access?

Reply:
No I don't :), just if you want to put it to the ocp-perfscale-cpt vault, pls help to update that information to the doc https://docs.google.com/document/d/1XL8kRey0M4hrlVyUnqPeGadVzFLgdMOP-IVWrsCX4ck

Reply:
Will do!!